glyph-brush/src/glyph_brush.rs

```diff
@@ -428,14 +428,15 @@ pub struct GlyphVertex {
 }
 
 /// Actions that should be taken after processing queue data
+#[derive(Debug, Clone, PartialEq)]
```

Comment:
I'd like to know the use case because I can imagine these causing issues. Firstly because equality & clone are not cheap for `Draw`. Equality checking seems strange, glyph-brush returns `ReDraw` if the last vertices are identical to this frame's, avoiding the need for that sort of check.

Reply:
But `Debug` is fine!